packages/dso-toolkit/reference/render/selectable--checkbox-nested.html

```diff
@@ -0,0 +1,40 @@
+<div class="dso-selectable">
+  <input type="checkbox" id="checkbox-nested-parent" name="selectable--checkbox-nested">
+  <label for="checkbox-nested-parent">
+    Selecteer mij
+  </label>
+  <div class="dso-selectable-children">
+    <div class="dso-selectable">
+      <input type="checkbox" id="checkbox-child-1" name="selectable--default">
+      <label for="checkbox-child-1">
+        Selecteer mij
+      </label>
+    </div>
+    <div class="dso-selectable">
+      <input type="checkbox" id="checkbox-child-2" name="selectable--default">
+      <label for="checkbox-child-2">
+        Selecteer mij
+      </label>
+      <div class="dso-selectable-children">
+        <div class="dso-selectable">
+          <input type="checkbox" id="checkbox-child-4" name="selectable--default">
```

Comment:
Hoe zat het ook alweer met checkboxen en identieke `name`s ?

Reply:
Ik heb rondgekeken en ben een aantal verschillende antwoorden tegen gekomen over het gebruik van identieke `name`s bij checkboxen. Ze worden gebruikt om de uiteindelijke values te groeperen, en in het eenvoudigste geval krijg je dan een array terug met de geselecteerde values, zie onder het kopje Multiple Choices:

https://html5-tutorial.net/forms/checkboxes/

Maar ik ben ook antwoorden tegen gekomen waarbij de values elkaar overschrijven en alleen de laatste door komt, dit zou vooral het geval zijn bij PHP applicaties.

@gofix @timveld hebben wij hier een afspraak over staan?